sdks/go/pkg/beam/core/runtime/graphx/v1/v1.pb.go

```diff
@@ -212,18 +212,19 @@ const (
 	Type_CONTEXT Type_Special = 2
 	Type_TYPE    Type_Special = 3
 	// Beam
-	Type_EVENTTIME     Type_Special = 10
-	Type_WINDOW        Type_Special = 22
-	Type_KV            Type_Special = 11
-	Type_COGBK         Type_Special = 13
-	Type_WINDOWEDVALUE Type_Special = 14
-	Type_T             Type_Special = 15
-	Type_U             Type_Special = 16
-	Type_V             Type_Special = 17
-	Type_W             Type_Special = 18
-	Type_X             Type_Special = 19
-	Type_Y             Type_Special = 20
-	Type_Z             Type_Special = 21
+	Type_EVENTTIME          Type_Special = 10
+	Type_WINDOW             Type_Special = 22
+	Type_KV                 Type_Special = 11
+	Type_COGBK              Type_Special = 13
+	Type_WINDOWEDVALUE      Type_Special = 14
+	Type_BUNDLEFINALIZATION Type_Special = 23
```

Comment:
Never manually update Generated Proto files:

You'll need to add the type to here, and regen the file. https://github.com/apache/beam/blob/5cfff1ac54b7446b0627f58a1e9cd5f9472d64c2/sdks/go/pkg/beam/core/runtime/graphx/v1/v1.proto#L113

Reply:
Oof - not sure how I missed that this was generated. 

I regenerated, if it is convenient to wait for that change to go in that's fine though.

Reply:
@lostluck thoughts on taking this now? It will cause a merge conflict, but its inevitable that one of our prs will run into that and this one is ready to go (I think, obviously feel free to review/disagree), and that one has been hanging around for a bit. Would be nice to doc/finish this

Reply:
I'm fine with adding it now, I was merely hoping the other PR would get in. Merge conflicts happen, and it's not a hard merge, especially since the PR simplifies the generation step anyway.